src/loaders/external_png/tvgPngLoader.cpp

```diff
@@ -97,6 +116,7 @@ bool PngLoader::read()
         free(buffer);
         return false;
     }
+    if (content) free((void*)content);
```

Comment:
this looks an unrelated change... does this correct?

Reply:
removed